frontend/README.md

```diff
@@ -18,6 +18,21 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Plop
+Plop is a generator which will create a skeleton of a new route or new component for you.
+
+To use Plop, from `frontend`:
+
+```bash
+# route
+plop route
```

Comment:
oh, looks like it's not in the package.json for this project. I'm assuming it works for you locally because you might have it installed globally on your machine.

Reply:
Can you update the readme with instructions on how to use this without having to install the package globally?

Reply:
Should be fixed in latest commit, please pull and re-review.

Reply:
I saw the same error. You need to run `npm install plop` or `npm install plop --save-dev` and then commit the package.json.

Reply:
Confirmed that `plop` works after installing it as a dependency via `npm i -D plop` (same as `npm install plop --save-dev`):

![Screenshot 2023-09-13 at 8 22 04 AM](https://github.com/8l-project-jam/tastemate/assets/6445410/ae80031f-2c55-4bad-abc3-7a846e947459)

Reply:
Thanks for pointing out the omission. Should be fixed as of latest commit, please re-review.

Reply:
@clairenguyen Will you please re-review when you get a chance? I think your requested changes (that I think I've made) are blocking merge. 🙏